web/elm/src/Build/StepTree/StepTree.elm

```diff
@@ -421,6 +427,9 @@ viewTree model tree =
         OnAbort { step, hook } ->
             viewHooked "abort" model step hook
 
+        OnError { step, hook } ->
+            viewHooked "error" model step hook
```

Comment:
This needs a corresponding `.hook-error` style here:

https://github.com/concourse/concourse/blob/444bd916ed46febe5b39809a94d8ecdb90730ca0/web/assets/css/modules.less#L58-L76

It should set the border color to `@base09` (orange).

Reply:
Missed, will fix